osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/PushSetupAlarmMappingTest.java

```diff
@@ -42,30 +54,55 @@ public void testPushSetupAlarmMapping() {
                 org.opensmartgridplatform.domain.core.valueobjects.smartmetering.PushSetupAlarm
                     .class);
 
-    // check mapping
     assertThat(pushSetupAlarmMapped).isNotNull();
+
+    // check mapping of SendDestinationAndMethod
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod()).isNotNull();
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod().getDestination()).isNotNull();
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod().getTransportService())
         .isNotNull();
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod().getMessage()).isNotNull();
 
-    // Only a SendDestinationAndMethod is mapped:
-    assertThat(pushSetupAlarmMapped.getLogicalName()).isNull();
-    assertThat(pushSetupAlarmMapped.getCommunicationWindow()).isNull();
-    assertThat(pushSetupAlarmMapped.getNumberOfRetries()).isNull();
-    assertThat(pushSetupAlarmMapped.getPushObjectList()).isNull();
-    assertThat(pushSetupAlarmMapped.getRandomisationStartInterval()).isNull();
-    assertThat(pushSetupAlarmMapped.getRepetitionDelay()).isNull();
-
-    // port and host are combined into destination. The converter sets
-    // default values for the other two variables of a
-    // SendDestinationAndMethod.
+    // port and host are combined into destination. The converter sets default values for the other
+    // two variables of a SendDestinationAndMethod.
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod().getDestination())
         .isEqualTo(DESTINATION);
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod().getTransportService().name())
         .isEqualTo(TRANSPORTSERVICETYPE.name());
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod().getMessage().name())
         .isEqualTo(MESSAGETYPE.name());
+
+    // check mapping of PushObjectList
+    assertThat(pushSetupAlarmMapped.getPushObjectList()).isNotNull();
```

Comment:
Is this `isNotNull` check needed, I think it will be tested in the `hasSize` assertion as well?